kafka-workitem/src/main/java/org/jbpm/process/workitem/kafka/KafkaWorkItemHandler.java

```diff
@@ -109,22 +112,10 @@ public void executeWorkItem(WorkItem workItem,
             String key = (String) workItem.getParameter("Key");
             String value = (String) workItem.getParameter("Value");
 
-            try {
-
-                producer.send(new ProducerRecord(topic,
-                                                 key,
-                                                 value));
-                results.put(RESULTS_VALUE,
-                            "success");
-                manager.completeWorkItem(workItem.getId(),
-                                         results);
-            } catch (Exception e) {
-                LOG.error("Kafka error",
-                          e);
-                producer.flush();
-                results.put(RESULTS_VALUE,
-                            "failure");
-            }
+            Future<RecordMetadata> future = producer.send(new ProducerRecord(topic, key, value));
+            RecordMetadata metadata = future.get(5, TimeUnit.SECONDS);
+            
+            manager.completeWorkItem(workItem.getId(), null);
```

Comment:
```suggestion
            manager.completeWorkItem(workItem.getId(), Collections.singletonMap(RESULTS_VALUE, "success"));
```

Reply:
Added again, and also covered in a unit test to check the right value
Thanks for review @elguardian !